src/Functions/DateTimeTransforms.h

```diff
@@ -136,6 +140,14 @@ struct ToStartOfDayImpl
         auto date_time = time_zone.fromDayNum(ExtendedDayNum(d));
         return date_time < 0xffffffff ? date_time : time_zone.toDate(0xffffffff);
     }
+    static inline DecimalUtils::DecimalComponents<DateTime64> execute_extended_result(const DecimalUtils::DecimalComponents<DateTime64> & t, const DateLUTImpl & time_zone)
```

Comment:
Costmetic: My local clang format complains about case style. Should have been `executeExtendedResult`.

Reply:
Yes, it's even in [recommendations](https://clickhouse.com/docs/en/development/style/#names):

> For the names of functions (methods), use camelCase beginning with a lowercase letter.

So I renamed the functions as needed.